<html>
<head>
<title>TestNG:  Navegacion_Header1</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Navegacion_Header1</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>7/2/0</td>
</tr><tr>
<td>Started on:</td><td>Wed Nov 08 20:05:07 CET 2017</td>
</tr>
<tr><td>Total time:</td><td>54 seconds (54191 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='Salle.TestClasses.SalleHeader.Empresas()'><b>Empresas</b><br>Test class: Salle.TestClasses.SalleHeader</td>
<td><div><pre>java.lang.AssertionError: expected [2] but found [1]
	at Salle.TestClasses.SalleHeader.Empresas(SalleHeader.java:98)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1415300010", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1415300010'><pre>java.lang.AssertionError: expected [2] but found [1]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at Salle.TestClasses.SalleHeader.Empresas(SalleHeader.java:98)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</pre></div></td>
<td>2</td>
<td>Salle.TestClasses.SalleHeader@75437611</td></tr>
<tr>
<td title='Salle.TestClasses.prueba.Empresas()'><b>Empresas</b><br>Test class: Salle.TestClasses.prueba</td>
<td><div><pre>java.lang.AssertionError: expected [2] but found [1]
	at Salle.TestClasses.prueba.Empresas(prueba.java:81)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1110827725", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1110827725'><pre>java.lang.AssertionError: expected [2] but found [1]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at Salle.TestClasses.prueba.Empresas(prueba.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</pre></div></td>
<td>1</td>
<td>Salle.TestClasses.prueba@663c9e7a</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='Salle.TestClasses.SalleHeader.Alumni()'><b>Alumni</b><br>Test class: Salle.TestClasses.SalleHeader</td>
<td></td>
<td>4</td>
<td>Salle.TestClasses.SalleHeader@75437611</td></tr>
<tr>
<td title='Salle.TestClasses.prueba.Alumni()'><b>Alumni</b><br>Test class: Salle.TestClasses.prueba</td>
<td></td>
<td>4</td>
<td>Salle.TestClasses.prueba@663c9e7a</td></tr>
<tr>
<td title='Salle.TestClasses.SalleHeader.Alumnos()'><b>Alumnos</b><br>Test class: Salle.TestClasses.SalleHeader</td>
<td></td>
<td>7</td>
<td>Salle.TestClasses.SalleHeader@75437611</td></tr>
<tr>
<td title='Salle.TestClasses.prueba.Alumnos()'><b>Alumnos</b><br>Test class: Salle.TestClasses.prueba</td>
<td></td>
<td>8</td>
<td>Salle.TestClasses.prueba@663c9e7a</td></tr>
<tr>
<td title='Salle.TestClasses.SalleHeader.NuevosAlumnos()'><b>NuevosAlumnos</b><br>Test class: Salle.TestClasses.SalleHeader</td>
<td></td>
<td>10</td>
<td>Salle.TestClasses.SalleHeader@75437611</td></tr>
<tr>
<td title='Salle.TestClasses.prueba.NuevosAlumnos()'><b>NuevosAlumnos</b><br>Test class: Salle.TestClasses.prueba</td>
<td></td>
<td>10</td>
<td>Salle.TestClasses.prueba@663c9e7a</td></tr>
<tr>
<td title='Salle.TestClasses.SalleHeader.Universidades()'><b>Universidades</b><br>Test class: Salle.TestClasses.SalleHeader</td>
<td></td>
<td>1</td>
<td>Salle.TestClasses.SalleHeader@75437611</td></tr>
</table><p>
</body>
</html>